epochStart/metachain/systemSCs.go

```diff
@@ -1125,7 +1126,7 @@ func (s *systemSCProcessor) getArgumentsForSetOwnerFunctionality(userValidatorAc
 		}
 	}
 
-	err = common.GetErrorFromChanNonBlocking(leavesChannels.ErrChan)
```

Comment:
i think `GetErrorFromChanNonBlocking` from common is not used anymore, maybe we can remove it

Reply:
Removed.